README.md

```diff
@@ -40,3 +40,25 @@ To compile and run the program, use `cabal run exes --` followed by the followin
   - privatekey - *local filepath for ssh private key*
   - script     - *local filepath of `.pbs` script to accomany the `qsub` command*
   - logFile    - *remote filepath of logfile produced by `.pbs` script to copy back to local system*
+
+## Testing in CI
+
+Integration tests are run against a version of OpenPBS dockerised along with an openssh server.
+The code for building the docker image (and associated scripts) are in the `ci` directory.
+It is tricky to build this image on aarch64-darwin as building the docker image involves compiling OpenPBS from source.
+
+The `.github/workflows/build-pbs.yml` workflow file builds the image in ci (only if there has been a change to code in the `ci` directory, or the `build-pbs.yml` workflow file) and pushes the image to a GCP artifact registry.
+The `.github/workflows/push.yml` builds a fully statically linked version of `hpci`, pulls the test docker image, and runs the new `hpci` binary to connect to the OpenPBS docker container and run a basic job submission.
```

Comment:
If renaming CI, should change this too

Reply:
Addressed in https://github.com/Garvan-Data-Science-Platform/hpci/pull/3/commits/d9e3f864b7c9cf93986aa6c4ccbf168ba9eac07d